### DISCUSSION TITLE
accessibility

### DISCUSSION THREAD

**Discussion by muzala** [community/community#65038] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

this is so easy to get protected from the system you can make your 2FA and for your users very easy

**Comment by ghostinhershell** [comment]
Hi @muzala, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by muzala** [reply]
thank you so much for the update and if course most of the time i get some difficulties issue that i cannot had chance to have a solution.and the problem is like the dependencies block me in some part of my work thanks

**Reply by ghostinhershell** [reply]
@muzala To be clear, are you running into issues using [2 factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa)?

**Reply by muzala** [reply]
No i was just saying that you got yours and you can open one for your users

**Reply by muzala** [reply]
Hi gostshell how are you please tell me did you tried using the kubernete for some time?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @muzala! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by muzala** [reply]
thank you all the best
